MIGRATION.md

```diff
@@ -15,11 +18,70 @@ The catalog has its own version number, as we may not require to update it with
 
 Version `1.0` of the catalog adds an important property to models `backend`, defining the type of framework required by the model to run (E.g. LLamaCPP, WhisperCPP).
 
-### Changelog
+### 🛠️ How to migrate
+
+You can either delete any existing `user-catalog` by deleting the `~/.local/share/containers/podman-desktop/extensions-storage/redhat.ai-lab/user-catalog.json`.
+
+> [!WARNING]  
+> This will remove the models you have imported from the catalog. You will be able to import it again afterward.
+
+If you want to keep the data, you can migrate it by updating certain properties.
+
+### Recipes
+
+The recipe object has a new property `backend` which define which framework is required. 
+Value accepted can be `llama-cpp`, `whisper-cpp`, `none`.
+
+Moreover, the `models` property has been changed to `recommended`.
```

Comment:
We should explain the change of semantic between models and recommended and what is happening when recommended is absent

Reply:
:+1: